ownCloud/Settings/SettingsViewController.swift

```diff
@@ -0,0 +1,37 @@
+//
+//  SettingsViewController.swift
+//  ownCloud
+//
+//  Created by Pablo Carrascal on 30/04/2018.
+//  Copyright © 2018 ownCloud GmbH. All rights reserved.
+//
+
+/*
+ * Copyright (C) 2018, ownCloud GmbH.
+ *
+ * This code is covered by the GNU Public License Version 3.
+ *
+ * For distribution utilizing Apple mechanisms please see https://owncloud.org/contribute/iOS-license-exception/
+ * You should have received a copy of this license along with this program. If not, see <http://www.gnu.org/licenses/gpl-3.0.en.html>.
+ *
+ */
+
+import UIKit
+
+class SettingsViewController: StaticTableViewController {
+
+    override func viewDidLoad() {
+        super.viewDidLoad()
+
+        self.navigationItem.title = "Settings".localized
+
+        let standardUserDefaults = UserDefaults.standard
```

Comment:
IIRC we'll need to grab a different UserDefaults instance via UserDefaults(suiteName: appGroupIdentifier) here. However, since it'll be needed in more places, I'd really like to add that to OCAppIdentity, so SettingsViewController can grab it from there in the future. 

For now, please use
```swift
// TODO: Use OCAppIdentity-provided user defaults in the future
let userDefaults = UserDefaults(suitename: OCAppIdentity.sharedAppIdentity().appGroupIdentifier)
```